code/modules/mob/living/simple_animal/friendly/robot_customer.dm

```diff
@@ -0,0 +1,52 @@
+///Robot customers
+/mob/living/simple_animal/robot_customer
+	name = "robot"
```

Comment:
```suggestion
	name = "space-tourist bot"
```

Reply:
i'll be using the datums to dynamically rename the robots in the future